oioioi/urls.py

```diff
@@ -44,7 +44,8 @@
             urls_module = import_module(app + '.urls')
             if hasattr(urls_module, 'urlpatterns'):
                 urlpatterns += getattr(urls_module, 'urlpatterns')
-        except ImportError:
+        except ImportError as e:
+            print(app, e)
```

Comment:
Please remove this change from this PR, we should resolve this issue in separate Pull Request